common/_overrides.scss

```diff
@@ -7,20 +7,44 @@
     background-color: $hover-bg !important;
     color: $hover-color !important;
     border-color: $hover-border-color !important;
+    & span {
+      color: $hover-color !important;
+    }
+  }
+
+  & span {
+    color: $color !important;
   }
 }
 
 // Set the default color of links
-a:not(.btn),
```

Comment:
I'm not sure about the implications of this. Isn't it a breaking change for the legacy theme?

Reply:
I have not seen problem in legacy experience, but it seems to me that in the way it was introduced, probably it is not working as intended, as the a:not(.help-link) will overwrite the style of a:not(.btn) previously defined and as consequence anchor with .btn class would be colored with $link-color, unless defined otherwise(overwrited with other classes)

Reply:
Ah, I haven't noticed that we're merging these two rules. All good then :+1: